api/src/main/java/org/pmiops/workbench/db/dao/UserService.java

```diff
@@ -80,8 +80,29 @@ void setTwoFactorAuthBypassTime(
   void logAdminWorkspaceAction(
       long targetWorkspaceId, String targetAction, Object oldValue, Object newValue);
 
+  /**
+   * Find users with Registered Tier access whose name or username match the supplied search terms.
+   *
+   * @param term User-supplied search term
+   * @param sort Option(s) for ordering query results
+   * @return the List of DbUsers which meet the search and access requirements
+   * @deprecated use {@link #findUsersBySearchString(java.lang.String, java.lang.String,
+   *     org.springframework.data.domain.Sort)} instead.
+   */
+  @Deprecated
   List<DbUser> findUsersBySearchString(String term, Sort sort);
```

Comment:
Modify the currently-in-use version to have the same behavior: only find Registered users